content/github/building-a-strong-community/creating-a-default-community-health-file.md

```diff
@@ -44,7 +44,7 @@ You cannot create a default license file. License files must be added to individ
   ![Owner drop-down menu](/assets/images/help/repository/create-repository-owner.png)
 3. Type **.github** as the name for your repository, and an optional description.
   ![Create repository field](/assets/images/help/repository/default-file-repository-name.png)
-4. Choose to make the repository public.
+4. Make sure the repository status is set to "public" (a repository for default files cannot be private).
```

Comment:
The text changes look good 💖 

Since "public" is a UI element that you can select, it looks as if it should be bold text, so I'd suggest:

```suggestion
4. Make sure the repository status is set to **Public** (a repository for default files cannot be private).
```

For reference: https://github.com/github/docs/blob/main/contributing/content-style-guide.md#radio-buttons

Reply:
Otherwise, this looks ready to merge 😄

Reply:
Done! :)